samples/getting-started/android/app/build.gradle

```diff
@@ -35,6 +30,29 @@ android {
     }
 }
 
+task buildSwiftArm64(type: Exec) {
+    commandLine 'sh', '-c', getSwiftBuildCommand('arm64-v8a')
+}
+
+task buildSwiftIntel64(type: Exec) {
+    commandLine 'sh', '-c', getSwiftBuildCommand('x86_64')
+}
+
+task cleanSwift(type: Exec) {
+    commandLine 'sh', '-c', "rm -rf ../../build"
+}
+
+preBuild.dependsOn buildSwiftArm64
+preBuild.dependsOn buildSwiftIntel64
+
+clean.dependsOn cleanSwift
+
+def getSwiftBuildCommand(abi) {
+    def cmakeBuildType = "RelWithDebInfo"
+    def buildScript = "../../../../swift-android-toolchain/swift-build.sh"
+    return "CMAKE_BUILD_TYPE=$cmakeBuildType ANDROID_ABI=$abi LIBRARY_OUTPUT_DIRECTORY='${projectDir}/src/main/jniLibs/${abi}' $buildScript ../../"
+}
```

Comment:
calling cmake from gradle instead inside the other cmake seems easier to grasp 👍  is there any other advantage?

Reply:
we mainly changed this because with the "calling cmake from cmake" approach we sometimes were missing the libs in the final apk for some weird reason. I think this is a bug in the gradle tools somewhere. It never happened with the new approach because explicitly build the libs on `preBuild`

Reply:
another benefit is that we now see all swift build logs and errors. with the old approach it was usually hidden behind "building cmake..."